onebusaway-android/src/google/java/org/onebusaway/android/map/googlemapsv2/VehicleOverlay.java

```diff
@@ -300,9 +300,10 @@ public void onInfoWindowClick(Marker marker) {
         ObaTripStatus status = mMarkerData.getStatusFromMarker(marker);
 
         if (mController != null && mController.getFocusedStopId() != null) {
-            TripDetailsActivity.start(mActivity, status.getActiveTripId(), mController.getFocusedStopId());
+            TripDetailsActivity.start(mActivity, status.getActiveTripId(),
+                    mController.getFocusedStopId(), "vehicle");
```

Comment:
"vehicle" string should be moved to a constant `SCROLL_MODE_VEHICLE` in `TripDetailsListFragment`.